src/diffusers/models/adapter.py

```diff
@@ -51,24 +51,26 @@ def __init__(self, adapters: List["T2IAdapter"]):
         if len(adapters) == 1:
             raise ValueError("For a single adapter, please use the `T2IAdapter` class instead of `MultiAdapter`")
 
-        # The outputs from each adapter are added together with a weight
-        # This means that the change in dimenstions from downsampling must
-        # be the same for all adapters. Inductively, it also means the total
-        # downscale factor must also be the same for all adapters.
-
-        first_adapter_total_downscale_factor = adapters[0].total_downscale_factor
-
+        # The outputs from each adapter are added together with a weight.
+        # This means that the change in dimensions from downsampling must
+        # be the same for all adapters. Inductively, it also means the
+        # downscale_factor and total_downscale_factor must be the same for all
+        # adapters.
         for idx in range(1, len(adapters)):
-            adapter_idx_total_downscale_factor = adapters[idx].total_downscale_factor
-
-            if adapter_idx_total_downscale_factor != first_adapter_total_downscale_factor:
+            if (
+                adapters[idx].total_downscale_factor != adapters[0].total_downscale_factor
```

Comment:
Could we try to assign `adapters[0].total_downscale_factor` in a new variable as was before? I think that slightly improves on the readability front.

Reply:
Cleaned up in [1ca2532](https://github.com/huggingface/diffusers/pull/5435/commits/1ca2532b940f6e8f87e224325093d1ef0278065f)